### ISSUE TITLE
AttributeError: 'Nodes' object has no attribute 'status'

### ISSUE THREAD

**Issue by simplextech** [shbatm/hacs-isy994#6]
> 2019-12-17 16:40:16 ERROR (MainThread) [homeassistant.setup] Error during setup of component isy994
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 172, in _async_setup_component
    hass, processed_config
  File "/config/custom_components/isy994/__init__.py", line 474, in async_setup
    log=_LOGGER,
  File "/usr/local/lib/python3.7/site-packages/PyISY/isy.py", line 82, in __init__
    self.nodes = Nodes(self, xml=self.conn.get_nodes())
  File "/usr/local/lib/python3.7/site-packages/PyISY/Nodes/__init__.py", line 89, in __init__
    self.parse(xml)
  File "/usr/local/lib/python3.7/site-packages/PyISY/Nodes/__init__.py", line 288, in parse
    Group(self, nid, nname, members, controllers),
  File "/usr/local/lib/python3.7/site-packages/PyISY/Nodes/group.py", line 33, in __init__
    for m in self.members
  File "/usr/local/lib/python3.7/site-packages/PyISY/Nodes/group.py", line 33, in <listcomp>
    for m in self.members
AttributeError: 'Nodes' object has no attribute 'status'

Latest version of hass.io and hacs integration.  I get this PyISY error from my primary ISY which has several Polyglot Nodeservers running.  From a secondary ISY with no nodeservers the component loads fine.

**Comment by shbatm** [comment]
@simplextech are you still having this error. I have not been able to replicate it. The only thing I suspect is it may be related to grouping and automicus/PyISY#11 or a specific node server. I have not received any errors with my nodeservers (Harmony Hub, MQTT, NodeLink).

**Comment by simplextech** [comment]
I was having the error.  I spent a lot of time trying to figure out what it could be and I couldn't narrow it down.  I suspect you are correct in it was some grouping or rather some corruption of grouping information.  What I ultimately did was I completely reset/wiped my ISY and manually added each and every device back and recreated scenes etc.  Each one I restarted HASS to check and the error was never replicated.  I have all of my devices and scenes all back in the ISY as they were before yet now everything works.... I suspect it was some underlying corruption that did not cause any obvious problems with just the ISY but it did with PyISY. After the rebuild of the ISY and devices everything is now working fine.
